server/routes/standardRouter.ts

```diff
@@ -12,7 +13,8 @@ export function standardRouter(): Router {
   router.use(auth.authenticationMiddleware())
   router.use(populateCurrentUser())
 
-  // check my diary 2FA specific session expiry
+  // CMD 2FA functionality - only if user hasn't gone through HMPPS Auth 2FA
+  router.use('/auth', loginRouter)
```

Comment:
`authenticationMiddleware` had been moved to below the `loginRouter` which meant that the `loginRouter` was handling the users not being signed in when it shouldn't have been.  Moved back below the middleware now.